lib/fix-google-html.js

```diff
@@ -255,6 +255,28 @@ export function unwrapLineBreaks (node) {
   node.children = children;
 }
 
+/**
+ * Moves linebreaks outside of anchor elements,
+ * if the linebreak is the first and/or last child of the anchor.
+ * @param {RehypeNode} node 
+ */
+export function moveLinebreaksOutsideOfAnchors (node) {
+  visit(node, 'element', (node, index, parent) => {
+    console.log(node)
+    if (node.tagName === 'a') {
+      const children = node.children;
+      if (children.length > 1 && children[children.length - 1].tagName === 'br') {
+        const endingBr = children.pop()
+        parent.children.splice(index + 1, 0, endingBr);
+      }
+      if (children.length > 0 && children[0].tagName === 'br') {
```

Comment:
Same as above for you this could be simplified:

```suggestion
      if (children[0]?.tagName === 'br') {
```